base/applications/rapps/unattended.cpp

```diff
@@ -43,12 +42,12 @@ BOOL UseCmdParameters(LPWSTR lpCmdLine)
         INFCONTEXT Context;
         if (SetupFindFirstLineW(InfHandle, L"RAPPS", L"Install", &Context))
```

Comment:
This branch is for passing a .inf (or .ini) file to rapps

Reply:
(and most likely what inspired the filename)

Reply:
hmm which filename?

Reply:
unattend.cpp